samples/sensor/bh1750/src/main.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2022 morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/sensor.h>
+
+/*
+ * Get a device structure from a devicetree node with compatible
+ * "rohm,bh1750". (If there are multiple, just pick one.)
+ */
+static const struct device *get_bh1750_device(void)
+{
+	const struct device *dev = DEVICE_DT_GET_ONE(rohm_bh1750);
```

Comment:
We want to stop adding new driver-specific sensor samples for every new sensor driver. Please review the generic accel and mag samples (`samples/sensor/accel_polling` and `samples/sensor/magn_polling`), and update this sample similarly.